### PR TITLE
Changed translation of "loading"

### PR DESCRIPTION
**Pull request by bramhaag**
Changed the value of "loading" from "Laden" to "Loading" in the www/locales/en.json file

### CODE REVIEW

**Comment by gkrol-zermelo** [comment]
Thanks Bram. This pull request also contains a commit with comment "SlideView swipw test". That is not supposed to be included I suppose? About the swiping functionality: we had that in the beginning but it interfered with scrolling the schedule vertically so we removed it. Something to consider when adding it back in.

**Comment by bramhaag** [comment]
I was looking though the code when I saw some code that was commented out. I though this was for the swiping functionallity and since I prefer swiping to reveal the navigation bar on the right I decided to test it out. It was after I created the pull request, but I didn't know it would update the pull request.

TL; DR: The "SlideView swipe test" commit wasn't supposed to be in this pull request

**Comment by gkrol-zermelo** [comment]
Yeah, it's weird that it updates the pull request. I've changed the translation manually. Thanks!
